disclosure_wrangler.py

```diff
@@ -23,35 +22,42 @@ class EnvironSchema(Schema):
     sns_topic_arn = fields.Str(required=True)
     sqs_message_group_id = fields.Str(required=True)
     sqs_queue_url = fields.Str(required=True)
+    csv_file_name = fields.Str(required=True)
 
 
 def lambda_handler(event, context):
     """
-    Main entry point into wrangler
-    :param event: json payload containing:
-        RuntimeVariables:{
-            disclosivity_marker: The name of the column to put 'disclosive' marker.
-            publishable_indicator: The name of the column to put 'publish' marker.
-            explanation: The name of the column to put reason for pass/fail.
-            total_column: The name of the column holding the cell total.
-            top1_column: The name of the column largest contributor to the cell.
-            top2_column: The name of the column second largest contributor to the cell.
-        }
+    Responsible for executing specified disclosure methods, masking values which could
+    be used to identify specific responders.
+    :param event: JSON payload containing:
+    RuntimeVariables:{
+        disclosivity_marker: The name of the column to put 'disclosive' marker.
+        publishable_indicator: The name of the column to put 'publish' marker.
+        explanation: The name of the column to put reason for pass/fail.
+        total_column: The name of the column holding the cell total.
+        parent_column: The name of the column holding the count of parent company.
+        threshold: The threshold used in the disclosure steps.
+        cell_total_column: The name of the column holding the cell total.
+        top1_column: The name of the column largest contributor to the cell.
+        top2_column: The name of the column second largest contributor to the cell.
+        stage5_threshold: The threshold used in the disclosure calculation.
+        disclosure_stages: The stages of disclosure you wish to run e.g. (1 2 5)
+    }
     :param context: AWS Context Object.
-    :return: Success - True/False & Checkpoint
+    :return final_output: Dict containing either:
+        {"success": True, "data": <stage 2 output - Type: JSON >}
```

Comment:
Why stage 2?

Reply:
That was left over from a copy + paste job... I noticed tho that it actually returns a checkpoint and not data so I've now changed it to reflect.